lib/wor/batchifier.rb

```diff
@@ -12,16 +12,25 @@ module Batchifier
     # => optionaly a batch_size and a merge strategy. It will slice the collection and
     # => apply the chozen strategy to all chunks and merge the results. It expects the responses
     # => to be Hash. It can ignore them if the given strategy is no_response
-    def execute_in_batches(collection, batch_size: 100, strategy: :add)
+    def execute_in_batches(collection, batch_size: 100, strategy: :add, &block)
       strategy_class = classify_strategy(strategy)
       collection.lazy.each_slice(batch_size).inject(strategy_class.new.base_case) do |rec, chunk|
-        response = yield(chunk)
+        response = batch_process(chunk, strategy, &block) if block_given?
```

Comment:
What if there is are no blocks given for processing the batch? This escapes the scope of this PR but I'll add this as an issue to raise an exception on incorrect usage of the gem.